spyder/plugins/editor/widgets/recover.py

```diff
@@ -99,6 +99,8 @@ def __init__(self, autosave_mapping, parent=None):
         self.add_table()
         self.add_cancel_button()
         self.setWindowTitle(_('Recover from autosave'))
+        self.setFixedSize(670, 400)
```

Comment:
Any particular reason for this fixed size? should we make it resizable?

Reply:
Basically I use this fixed size to show all the table horizontally and to prevent the dialog to get too tall, but maybe I can set the width to the width of the table instead of a fixed number. About the resize I think @ccordoba12 told me to prevent it

Reply:
Basically I use this fixed size to show all the table horizontally and to prevent the dialog to get too tall, but maybe I can set the width to the width of the table instead of a fixed number. About the resize I think @ccordoba12 told me to prevent it

Reply:
It is ok, we can merge as is!